scipp-imaging-2D.ipynb

```diff
@@ -0,0 +1,498 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to start\n",
+    "\n",
+    "Before starting you must:\n",
+    "- Ensure that `scipp` and `mantid` are on your `PYTHONPATH`.\n",
```

Comment:
These nice instructions use of config etc are not mirrored in the benchmark ipynb

Reply:
Good catch. I've updated it to point to the non-bench notebook for instructions as I don't want to duplicate those, because they will get out of date. I've also added a link to the scipp issue with previous benchmarks.